http/src/main/resources/reference.conf

```diff
@@ -577,6 +627,25 @@ gke {
     minMemoryGb = 5
     minNumOfCpus = 3
     enabled = true
+    # App developers - Please keep the list of non-backward compatible versions in the list below
+    chartVersionsToExcludeFromUpdates = [
```

Comment:
Curious if there's a way to disable updates for a given app type? E.g. say we want to prevent people upgrading Galaxy for now..

Reply:
There is no specific flag to disable, for now I simply added all versions to the exclude list 😬